pages/embedded-portal.tsx

```diff
@@ -26,20 +26,33 @@ import StepList, { Step } from "../components/shared/step-list";
 import Workspace from "../components/embedded-portal/workspace";
 import { theme } from "../lib/theme";
 import { useFlashMessages, useOnClickOutside } from "../lib/hooks/usehooks";
+import { Flatfile } from "@flatfile/api";
+import { WorkflowType, getSpace, getWorkbook } from "../lib/flatfile";
+
+type WorkbookConfig = {
+  name: string;
+  labels?: string[];
+  spaceId: Flatfile.SpaceId;
+  environmentId: Flatfile.EnvironmentId;
+  /** Sheets must be <= 50 */
```

Comment:
why < 50?

Reply:
Pasted from the docs. Removing the comment